benchmarks/cugraph/standalone/bulk_sampling/run_sampling.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+# Copyright (c) 2024, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+conda init
+source ~/.bashrc
+conda activate rapids
+
+BATCH_SIZE=$1
+FANOUT=$2
+REPLICATION_FACTOR=$3
+SCRIPTS_DIR=$4
+NUM_EPOCHS=$5
+
+SAMPLES_DIR=/samples
```

Comment:
Assuming we are mounting this to the most performant path.

Reply:
Yes, it is up to us to set `LOGS_DIR`, `SAMPLES_DIR`, and `DATASETS_DIR` in `run_train_job.sh` correctly.  In the `srun` command, those are mounted to `/logs`, `/samples`, and `/datasets` in the container that this script runs in.